src/PMGDQuery.cc

```diff
@@ -463,6 +463,8 @@ void PMGDQuery::UpdateNode(int ref,
                         const Json::Value& constraints,
                         bool unique)
 {
+    _readonly = false;
```

Comment:
This bug fix for UpdateNode should be merged into the commit where UpdateNode was added.